packages/flutter_web_plugins/lib/src/plugin_event_channel.dart

```diff
@@ -129,7 +132,7 @@ class _EventChannelHandler<T> {
 
   Future<ByteData> _listen() async {
     if (subscription != null) {
-      await subscription.cancel();
+      await subscription!.cancel();
     }
```

Comment:
Is awaiting on null allowed? If so, then this entire `if` block can be just:

```dart
await subscription?.cancel();
```

Reply:
Good point, I don't know!

Reply:
It seems you *can* await on null (it'll resolve to null):

```dart
void main() async {
  Future<int>? bah = null;
  print(await bah); // prints null
  print('done'); // prints done
}
```